README.rst

```diff
@@ -46,7 +46,7 @@ To run Django's test suite:
   ``docs/internals/contributing/writing-code/unit-tests.txt``, published online at
   https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/unit-tests/#running-the-unit-tests
 
-Supporting the Development of Django
+Supporting the Development of Django :green_heart:
```

Comment:
README should be readable not only in GitHub, so IMO emojis are unnecessary here.